### PR TITLE
JIRA: small bugfixes, improvements and cleanup

### PR DESCRIPTION
**Pull request by valentijnscholten**
- bugfix on `edit_engagement`
- set note author to be `jira` after pushing/linking to JIRA issue instead of `finding.reporter`. This is not perfect, but in celery we don't have the request object so we don't know who is actually pushing it.
- rename `add_issue` and `update_issue` in utils to more explicit `add_jira_issue` and `update_jira_issue`
- cleanup left over print/log statements from #2649 
- cleanup other print/log statements because they are not my favourite

### CODE REVIEW

File: dojo/tools/whitesource/parser.py [hunk 1]
```diff
@@ -67,7 +67,7 @@ def _build_common_output(node, lib_name=None):
                             topfix_node.get('fixResolution')
                         )
                 except Exception as e:
-                    print("Error handling topFix node. {}").format(e)
+                    ("Error handling topFix node. {}").format(e)
```
Comment (Maffooch) [inline dojo/tools/whitesource/parser.py:70]: Is a logger missing here?

Comment (valentijnscholten) [reply dojo/tools/whitesource/parser.py:70]: wow, nice spot. am struggling lately with visual studio code is it's so damn slow with python code that even copy and pasting is laggy

Comment (valentijnscholten) [reply dojo/tools/whitesource/parser.py]: I reverted it back to what it was.... which is invalid python code...

**Comment by valentijnscholten** [comment]
Merged this into dev as the other mods are on holidays and this is a lightweight PR but has a bugfix for dev (bug is not present in 1.7.x).

**Pull request merged**